completions/ip

```diff
@@ -357,7 +357,14 @@ _comp_cmd_ip()
                 all) ;;
                 *)
                     ((cword == subcword)) &&
-                        _comp_compgen -- -W 'help all'
+                        _comp_compgen_split -- "help all $(
+                            {
+                                $1 -c=never monitor help || $1 monitor help
+                            } 2>&1 | command sed -e \
+                                '/OBJECTS := /,/|.*$/!d' -e \
```

Comment:
I think `/|.*$/` matches any line containing `|`, so this condition fails when the number of lines of OBJECTS becomes three or more. For example, with this mock text:

```bash
$ printf '%s\n' 'OBJECTS := address | link |' '    mroute | neigh |' '    nexthop | stats'
OBJECTS := address | link |
    mroute | neigh |
    nexthop | stats
```

The range only selects the first two:

```bash
$ printf '%s\n' 'OBJECTS := address | link |' '    mroute | neigh |' '    nexthop | stats' | sed -ne '/OBJECTS := /,/|.*$/p'
OBJECTS := address | link |
    mroute | neigh |
$
```

Reply:
Nicely spotted, I changed it to match until a line that don't end with with `|`.

Reply:
Thanks!